app/helpers/vacancies_helper.rb

```diff
@@ -180,8 +180,8 @@ def vacancy_working_patterns(vacancy)
     return vacancy_working_patterns_summary(vacancy) unless vacancy.full_time_details? || vacancy.part_time_details?
 
     safe_join [
-      (tag.li { t("jobs.full_time_details", details: vacancy.full_time_details) } if vacancy.full_time_details?),
-      (tag.li { t("jobs.part_time_details", details: vacancy.part_time_details) } if vacancy.part_time_details?),
+      (tag.div { t("jobs.full_time_details", details: vacancy.full_time_details) } if vacancy.full_time_details?),
```

Comment:
if this changes from a list element, you want to remove the wrapping `ul` element

Reply:
there is no wrapping ul element, thats the point

Reply:
I see the wrapping element

https://github.com/DFE-Digital/teaching-vacancies/blob/main/app/views/publishers/vacancies/_vacancy_review_sections.html.slim#L43

Reply:
hmmm i dont, will investigate why this doesnt render properly

![Screenshot 2022-09-12 at 09 50 36](https://user-images.githubusercontent.com/1792451/189612542-e6cfa808-9a13-443c-834d-c305ce1be2f6.png)